src/openforms/authentication/models.py

```diff
@@ -0,0 +1,77 @@
+from django.contrib.auth.hashers import make_password as get_salted_hash
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from openforms.contrib.kvk.validators import validate_kvk
+from openforms.utils.validators import validate_bsn
+
+from .constants import AuthAttribute
+
+
+# TODO: what about co-sign data?
```

Comment:
Let's leave it as is for now, but it should get a similar construct indeed. There's some other co-sign rework in the pipeline, we can bring this in when we're dealing with that.